src/algorithms/reco/PrimaryVerticesConfig.h

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2024 Daniel Brandenburg
+
+#pragma once
+
+#include <string>
+#include <DD4hep/DD4hepUnits.h>
+
+namespace eicrecon {
+
+  struct PrimaryVerticesConfig {
+
+    // For now these are wide open
+    // In the future the cut should depend
+    // on the generator settings
+    float maxVr = 50.0; // mm
+    float maxVz = 500.0; // mm
+    float maxChi2 = 10000.0; //
+    int   minNtrk = 1;  // >=
+    int   maxNtrk = 1000000; // <=
```

Comment:
These are post-reconstruction analysis-level selection cuts. The IVF algorithm cuts are embedded in the ACTS algorithm. They are different.

Reply:
Well that is the question, in what way they are different? And if they are, why do we prefer one over another?

I understand, we are discussing trivial facilities, but it would be nice to draw from ACTS experience where possible.

Reply:
Those ACTS cuts still apply in the IVF finding/fitting. The cuts listed here at analysis level if there is selection to filter out any vertices. The geometric parameters (i.e. Vr. Vz etc.) should be experiment specific. Other cuts are very loose at this moment, but we include the option here in case users want to apply tighter cuts (e.g. chi2 to remove bad fitted vertices).